src/frontend/interfaces/modal.interface.ts

```diff
@@ -0,0 +1,26 @@
+import { VariantType } from '@adminjs/design-system'
+
+export interface ConfirmModalData {
```

Comment:
file name should be `confirm-modal.interface.ts`

Reply:
changed